config-templates/shopify/order_line.yaml

```diff
@@ -0,0 +1,47 @@
+# ORDER_LINE CONFIG
+data_source:
+  name: shopify_orders_line
+  owners:
+    - support@transformdata.io
+  sql_query: |
+    select * from SHOPIFY.ORDER_LINE a left join SHOPIFY."ORDER" b on a.order_id = b.id
+  mutability:
+    type: immutable
+  identifiers:
+    - name: id
+      type: primary
+    - name: order
+      type: foreign
+      expr: order_id
+    - name: product_id
+      type: foreign
+    - name: variant_id
+      type: foreign
+    - name: origin_location_id
+      type: foreign
+    - name: destination_location_id
+      type: foreign
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: True
+        time_granularity: day
+      expr: created_at::date
+    - name: vendor
+      type: categorical
+    - name: title
+      type: categorical
+  measures:
+    - name: total_quantity
+      agg: sum
+      expr: quantity
+      create_metric: true
+    - name: avg_item_value
```

Comment:
i'm not sure this is the right name here - it is only the item average if the person is grouping by item - maybe just avg_price or value?